### DISCUSSION TITLE
Simple tag to differenciate issues for different pub packages

### DISCUSSION THREAD

**Discussion by FMorschel** [simolus3/drift#2860] [Ideas]
Hi! I'm here to ask so that (at least new issues or issues that are being worked on) could start receiving tags to differentiate them, tagging the pub package. 

I'm asking this because if someone wants to help, they can easily find the issues related to a specific package.

For example, if I were a really heavy sqlite programmer and not as familiar with postgres, or the other way around, I'd like to contribute to the package in a way I know how. And in my opinion, tags help to make categories a bit more clear.

**Comment by simolus3** [comment]
Thanks for the suggestion, I agree that this is useful. I have added a few tags for the different backends and components. I've applied them to the most recent issues and I'll try to remember also doing this for new ones.
